chore(ffmpeg-kit-react-native): update plugin to use AndroidConfig.Version.withBuildScriptExtMinimumVersion

thorbenprimke opened the pull request:
# Why

I referenced this plugin while working on #57. This just updates this plugin to use
AndroidConfig.Version's withBuildScriptExtMinimumVersion directly.

# How

Just updated code to use AndroidConfig.Version's withBuildScriptExtMinimumVersion
and deleted the local copy.

# Test Plan

Verified generated code

EvanBacon reviewed (approved):
nice work

EvanBacon reviewed (changes requested):
Tests aren't passing locally. Need to remove `withBuildScriptExtMinimumVersion-test.ts`, also be sure to validate everything looks good locally.

thorbenprimke commented:
> Tests aren't passing locally. Need to remove `withBuildScriptExtMinimumVersion-test.ts`, also be sure to validate everything looks good locally.

Yea, moved too fast.  Deleted the tests since they were the same as in the [config-plugins](https://github.com/expo/expo-cli/blob/main/packages/config-plugins/src/android/__tests__/Version-test.ts) package. 

Ran `yarn test` to ensure it's all good now.
